webpack.config.js

```diff
@@ -4,7 +4,6 @@ var fs = require('fs')
   , path = require('path')
   , webpack = require('webpack')
   , console = require('console')
-  , WrapperPlugin = require('wrapper-webpack-plugin')
```

Comment:
nitpick: this is still needed for the IE8 conditional comments - otherwise build error; np, i can add it.